src/validations.test.js

```diff
@@ -463,4 +464,73 @@ describe('validations', () => {
       expect(result).toEqual({ charLength: true, letter: true, digit: true });
     });
   });
+
+  describe('validateName', () => {
+    it('should not accept an empty name', () => {
+      // Arrange
+      const name = '';
+
+      // Act
+      const result = validateName(name);
+
+      // Assert
+      expect(result).toBeNull();
+    });
+
+    it('should not accept an name with one char', () => {
+      // Arrange
+      const name = 'a';
+
+      // Act
+      const result = validateName(name);
+
+      // Assert
+      expect(result).toEqual('validation_messages.error_invalid_name');
+    });
+
+    it('should accept an name with two chars', () => {
+      // Arrange
+      const name = 'Fe';
+
+      // Act
+      const result = validateName(name);
+
+      // Assert
+      expect(result).toBeNull();
+    });
+
+    it('should accept an name with more than one char in chinese language', () => {
+      // Arrange
+      const name = '陸軍上校';
+
+      // Act
+      const result = validateName(name);
+
+      // Assert
+      expect(result).toBeNull();
+    });
+
+    it('should not accept an name with special character', () => {
+      // Arrange
+      const name = 'Joh=n';
+
+      // Act
+      const result = validateName(name);
+
+      // Assert
+      expect(result).toEqual('validation_messages.error_invalid_name');
+    });
+
+    it('should return a custom error', () => {
+      // Arrange
+      const name = 'フェデ+';
```

Comment:
What do you think about avoiding weird character in order to make the cause of the error more visible?

Reply:
yes you right